packages/docs/src/theme/Footer/footer.json

```diff
@@ -0,0 +1,79 @@
+{
+  "footer": {
+    "contact": {
+      "logo_text": "Web3.Storage",
+      "prompt": "Sign up for free using your email address or GitHub.",
+      "cta": {
+        "url": "https://web3.storage/login",
+        "text": "SIGN UP",
+        "theme": "pink-blue"
+      }
+    },
+    "resources": {
+      "heading": "Resources",
+      "items": [
+        {
+          "text": "Quickstart Guide",
+          "url": "/#quickstart"
+        },
+        {
+          "text": "FAQ",
+          "url": "https://web3.storage/faq"
```

Comment:
nit: plz use at least domain relative urls throughout this file here so it's self-contained. I was clicking around a staging build of the docs site and ended up back on the production site, which gets confusing if the two aren't radically different.

I suspect there wont be a way in docusarus to have these be path relative urls, so they will break on path based ipfs gateways, but I can live with that.